graylog2-web-interface/src/views/components/visualizations/number/AutoFontSizer.jsx

```diff
@@ -16,45 +16,51 @@ type Props = {
 };
 
 const _multiplierForElement = (element, targetWidth, targetHeight) => {
+  const childSizeRatio = 0.8; // Propotion of the child size in relation to the container
```

Comment:
This constant could be pulled out of the function scope and into the module scope. Also: `Propotion` -> `Proportion`